FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,19 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 	
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+			
+			switch result {
+			case .failure:
+				completion(.failure(Error.connectivity))
+			case .success(let response):
+				guard response.1.statusCode == 200,
+					  let feedImages = try? JSONDecoder().decode(FeedImages.self, from: response.0) else {
+					return completion(.failure(Error.invalidData))
+				}
```

Comment:
This code can become more expressive by destructuring the `response` tuple:

```suggestion
			case let .success((data, response)):
				guard response.statusCode == 200,
					  let feedImages = try? JSONDecoder().decode(FeedImages.self, from: data) else {
					return completion(.failure(Error.invalidData))
				}
```
```suggestion
			case .success(let response):
				guard response.1.statusCode == 200,
					  let feedImages = try? JSONDecoder().decode(FeedImages.self, from: response.0) else {
					return completion(.failure(Error.invalidData))
				}
```